test.py

```diff
@@ -68,24 +66,31 @@ def test_semicolon(self):
     efg;3;3,0'''
         )
         c.execute('SELECT COUNT(heading_3) FROM %s' % DEFAULT_TABLE_NAME);
-        row = next(c)
```

Comment:
Again, why the change here or in other similar cases. Quite like the explicitness of `row = next(c)` ;-)